site/booster-2022.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: layouts/page.njk
+title: Booster 2022 abstracts
+description: To be or not to be, that is the question
+---
+
+## 30 min talk ideas
+
+### Building a healthy company culture
+
+When I rebuilt VIBBIO's product team from scratch in 2018, we made the conscious decision to embrace a fully remote setup. This gave us an advantage in recruiting but brought a significant change for the company.
+
+With a fully remote team, I could not get away with the all too pervasive management style of muddling through, I had to make deliberate choices about pratices and principles to foster the culture that I wanted to see.
+
+In this presentation, I'll talk about the challenges I faced, the lessons I learned, and give some insight into how good management principles can change companies for the better.
+
+### A primer on setting goals and how to measure progress
+
+We've probably all been there: someone hands us a KPI or any other kind of goal, and we just cannot make sense of it. Why this number, where did it come from, and why is this important now?
+
+I have been guilty of creating poor OKRs and KPIs as well as arrogantly dismissing goals that I didn't understand. Neither have been my proudest moments. To spare myself future embarrassment, I spent a non-trivial amount of time on picking apart the process and figuring out how the different levels of goals and metrics fit together.
+
+This presentation shares the results, so you can either set better goals yourself or ask better questions to the ones around you who do.
+
+## 10 min lightning talks
+
+### Why you can’t demand full autonomy without accepting full responsibility
+
+Autonomy has become the ultimate perk, the ultimate demand, and the ultimate expectation we software professionals place towards any job we consider. We usually do our best work when we have it.
+
+But are we as willing to accept the responsibility that comes with autonomy, as we are with demanding to make our own decisions?
+
+### Why it’s not hard to get respect from software engineers, and why this myth needs to die
+
+In a recent job interview, I was asked how I, as a non-engineer, managed to earn and maintain the respect of the engineers I work with. While I understand where the question comes from, there is a lot to unpack here.
+
+This lightning talk does just that: we'll pick apart the layers of hurt, misunderstanding, arrogance, and distrust on both sides of this statement.
+
+### Product owner is not a job
+
+
+
+### Good reasons and bad reasons to work for a startup
```

Comment:
IMO (having worked for startups) the pros/cons are not generic, and depend highly on the actual company. Not sure if that can be turned into something that applies to a wider audience.